src/lib/message/CHIPFabricState.cpp

```diff
@@ -263,7 +263,7 @@ CHIP_ERROR ChipFabricState::Init(GroupKeyStoreBase * groupKeyStore)
     ListenIPv4Addr = IPAddress::Any;
     ListenIPv6Addr = IPAddress::Any;
 
-#if defined(DEBUG) && !CHIP_SYSTEM_CONFIG_USE_LWIP
+#ifndef NDEBUG && !CHIP_SYSTEM_CONFIG_USE_LWIP
```

Comment:
`#if !defined(NDEBUG) && !CHIP_SYSTEM_CONFIG_USE_LWIP`

Would suggest using `!defined(NDEBUG) `everywhere for consistency reasons, since you have to anyway for nontrivial cases like this one.

Reply:
oh missed this one, that should have failed compile 🤔 . Anyway, fixed it. Thanks!